telegram/ext/_applicationbuilder.py

```diff
@@ -1092,6 +1092,13 @@ async def post_init(application: Application) -> None:
 
                 application = Application.builder().token("TOKEN").post_init(post_init).build()
 
+        Note:
+            If you implement custom logic that implies that you will **not** be using
+            :meth:`Application.run_polling` or :meth:`Application.run_webhook` to run
+            your application (like e.g. in
+            :any:`Custom Webhook Bot Example <examples.customwebhookbot>`),
+            make sure that you explicitly call this method.
```

Comment:
```suggestion
            :any:`Custom Webhook Bot Example <examples.customwebhookbot>`),.
```
There is not really a need to set the function on the `ApplicationBuilder` in this case. you can just manually do `await some_function(application)` if you're doing custom logic anyways :)

Can we use a sphinx insertion thingy to avoid repitition?

Reply:
Hmm...  makes sense :)

As for the insertion - it doesn't work if I mention methods there. Sphinx complains about it.

Reply:
ah, you'll have to use the full name then, see e.g. :meth:~telegram.ext.Application.run_polling

Reply:
OK, done. It didn't like the `:any:` link to the example either, so I had to replace it with an https hyperlink